diff --unified --recursive '--exclude=.pylint-ignores.patch' ./api_album.py /tmp/pylint_iHIz1e/api_album.py
--- ./api_album.py	2013-07-28 18:32:55.587158750 +0100
+++ /tmp/pylint_iHIz1e/api_album.py	2013-07-28 18:32:52.783144848 +0100
@@ -1,6 +1,6 @@
 from .objects import Album
 
-class ApiAlbums(object):
+class ApiAlbums(object): # pylint: disable=R0903,C0111
     def __init__(self, client):
         self._client = client
 
@@ -9,7 +9,7 @@
         results = self._client.get("/albums/list.json", **kwds)["result"]
         return [Album(self._client, album) for album in results]
 
-class ApiAlbum(object):
+class ApiAlbum(object): # pylint: disable=C0111
     def __init__(self, client):
         self._client = client
 
diff --unified --recursive '--exclude=.pylint-ignores.patch' ./api_photo.py /tmp/pylint_iHIz1e/api_photo.py
--- ./api_photo.py	2013-07-28 18:32:55.587158750 +0100
+++ /tmp/pylint_iHIz1e/api_photo.py	2013-07-28 18:32:52.787144864 +0100
@@ -17,7 +17,7 @@
             ids.append(photo)
     return ids
 
-class ApiPhotos(object):
+class ApiPhotos(object): # pylint: disable=C0111
     def __init__(self, client):
         self._client = client
 
@@ -51,7 +51,7 @@
             raise TroveboxError("Delete response returned False")
         return True
 
-class ApiPhoto(object):
+class ApiPhoto(object): # pylint: disable=C0111
     def __init__(self, client):
         self._client = client
 
diff --unified --recursive '--exclude=.pylint-ignores.patch' ./api_tag.py /tmp/pylint_iHIz1e/api_tag.py
--- ./api_tag.py	2013-07-28 18:32:55.587158750 +0100
+++ /tmp/pylint_iHIz1e/api_tag.py	2013-07-28 18:32:52.787144864 +0100
@@ -1,6 +1,6 @@
 from .objects import Tag
 
-class ApiTags(object):
+class ApiTags(object): # pylint: disable=R0903,C0111
     def __init__(self, client):
         self._client = client
 
@@ -9,7 +9,7 @@
         results = self._client.get("/tags/list.json", **kwds)["result"]
         return [Tag(self._client, tag) for tag in results]
 
-class ApiTag(object):
+class ApiTag(object): # pylint: disable=C0111
     def __init__(self, client):
         self._client = client
 
diff --unified --recursive '--exclude=.pylint-ignores.patch' ./config.py /tmp/pylint_iHIz1e/config.py
--- ./config.py	2013-07-28 18:32:55.587158750 +0100
+++ /tmp/pylint_iHIz1e/config.py	2013-07-28 18:32:52.787144864 +0100
@@ -1,7 +1,7 @@
 from __future__ import unicode_literals
 import os
 try:
-    from configparser import ConfigParser # Python3
+    from configparser import ConfigParser # Python3 # pylint: disable=F0401
 except ImportError:
     from ConfigParser import SafeConfigParser as ConfigParser # Python2
 try:
@@ -9,8 +9,8 @@
 except ImportError:
     import StringIO as io # Python2
 
-class Config(object):
-    def __init__(self, config_file, host,
+class Config(object): # pylint: disable=R0903,C0111
+    def __init__(self, config_file, host, # pylint: disable=R0913
                  consumer_key, consumer_secret,
                  token, token_secret):
         if host is None:
@@ -65,7 +65,7 @@
     parser = ConfigParser()
     parser.optionxform = str # Case-sensitive options
     try:
-        parser.read_file(buf) # Python3
+        parser.read_file(buf) # Python3 # pylint: disable=E1103
     except AttributeError:
         parser.readfp(buf) # Python2
 
diff --unified --recursive '--exclude=.pylint-ignores.patch' ./http.py /tmp/pylint_iHIz1e/http.py
--- ./http.py	2013-07-28 18:32:55.591158774 +0100
+++ /tmp/pylint_iHIz1e/http.py	2013-07-28 18:32:52.787144864 +0100
@@ -4,18 +4,18 @@
 import requests_oauthlib
 import logging
 try:
-    from urllib.parse import urlunparse # Python3
+    from urllib.parse import urlunparse # Python3 # pylint: disable=F0401,E0611
 except ImportError:
     from urlparse import urlunparse # Python2
 
 from .objects import TroveboxObject
-from .errors import *
+from .errors import * # pylint: disable=W0401
 from .config import Config
 
 if sys.version < '3':
-    TEXT_TYPE = unicode
+    TEXT_TYPE = unicode # pylint: disable=C0103
 else:
-    TEXT_TYPE = str
+    TEXT_TYPE = str # pylint: disable=C0103
 
 DUPLICATE_RESPONSE = {"code": 409,
                       "message": "This photo already exists"}
@@ -32,7 +32,7 @@
     This should be used to ensure that your application will continue to work
         even if the Trovebox API is updated to a new revision.
     """
-    def __init__(self, config_file=None, host=None,
+    def __init__(self, config_file=None, host=None, # pylint: disable=R0913
                  consumer_key='', consumer_secret='',
                  token='', token_secret='', api_version=None):
         self._api_version = api_version
diff --unified --recursive '--exclude=.pylint-ignores.patch' ./__init__.py /tmp/pylint_iHIz1e/__init__.py
--- ./__init__.py	2013-07-28 18:32:55.591158774 +0100
+++ /tmp/pylint_iHIz1e/__init__.py	2013-07-28 18:33:03.527198125 +0100
@@ -1,5 +1,5 @@
 from .http import Http
-from .errors import *
+from .errors import * # pylint: disable=W0401
 from ._version import __version__
 from . import api_photo
 from . import api_tag
@@ -19,7 +19,7 @@
     This should be used to ensure that your application will continue to work
         even if the Trovebox API is updated to a new revision.
     """
-    def __init__(self, config_file=None, host=None,
+    def __init__(self, config_file=None, host=None, # pylint: disable=R0913
                  consumer_key='', consumer_secret='',
                  token='', token_secret='',
                  api_version=None):
diff --unified --recursive '--exclude=.pylint-ignores.patch' ./main.py /tmp/pylint_iHIz1e/main.py
--- ./main.py	2013-07-28 18:32:55.591158774 +0100
+++ /tmp/pylint_iHIz1e/main.py	2013-07-28 18:32:52.787144864 +0100
@@ -23,7 +23,7 @@
 
 #################################################################
 
-def main(args=sys.argv[1:]):
+def main(args=sys.argv[1:]): # pylint: disable=R0912,C0111
     usage = "%prog --help"
     parser = OptionParser(usage, add_help_option=False)
     parser.add_option('-c', '--config', help="Configuration file to use",
@@ -81,13 +81,13 @@
             sys.exit(1)
 
     if options.method == "GET":
-        result = client.get(options.endpoint, process_response=False,
+        result = client.get(options.endpoint, process_response=False, # pylint: disable=W0142
                             **params)
     else:
         params, files = extract_files(params)
-        result = client.post(options.endpoint, process_response=False,
+        result = client.post(options.endpoint, process_response=False, # pylint: disable=W0142
                              files=files, **params)
-        for f in files:
+        for f in files: # pylint: disable=C0103
             files[f].close()
 
     if options.verbose:
diff --unified --recursive '--exclude=.pylint-ignores.patch' ./objects.py /tmp/pylint_iHIz1e/objects.py
--- ./objects.py	2013-07-28 18:32:55.591158774 +0100
+++ /tmp/pylint_iHIz1e/objects.py	2013-07-28 18:32:52.787144864 +0100
@@ -1,14 +1,14 @@
 try:
-    from urllib.parse import quote # Python3
+    from urllib.parse import quote # Python3 # pylint: disable=F0401,E0611
 except ImportError:
     from urllib import quote # Python2
 
 from .errors import TroveboxError
 
-class TroveboxObject(object):
+class TroveboxObject(object): # pylint: disable=R0903
     """ Base object supporting the storage of custom fields as attributes """
     def __init__(self, trovebox, json_dict):
-        self.id = None
+        self.id = None # pylint: disable=C0103
         self.name = None
         self._trovebox = trovebox
         self._json_dict = json_dict
@@ -54,7 +54,7 @@
         return self._json_dict
 
 
-class Photo(TroveboxObject):
+class Photo(TroveboxObject): # pylint: disable=C0111
     def delete(self, **kwds):
         """
         Delete this photo.
@@ -144,7 +144,7 @@
 
         self._replace_fields(new_dict)
 
-class Tag(TroveboxObject):
+class Tag(TroveboxObject): # pylint: disable=C0111
     def delete(self, **kwds):
         """
         Delete this tag.
@@ -165,7 +165,7 @@
         self._replace_fields(new_dict)
 
 
-class Album(TroveboxObject):
+class Album(TroveboxObject): # pylint: disable=C0111
     def __init__(self, trovebox, json_dict):
         self.photos = None
         self.cover = None
